src/rules/custom-media-pattern/index.js

```diff
@@ -0,0 +1,33 @@
+import { isRegExp } from "lodash"
+import {
+  report,
+  ruleMessages,
+  validateOptions
+} from "../../utils"
+
+export const ruleName = "custom-media-pattern"
+
+export const messages = ruleMessages(ruleName, {
+  rejected: `Custom media name does not match specified pattern`,
```

Comment:
Any chance we can use "expected" here? The message in `custom-property-pattern` was an oversight, which I've just corrected in #328.

I'm a stickler for consistency when it comes to user experience :)